base/inc/AdePT/LoopNavigator.h

```diff
@@ -128,6 +128,10 @@ class LoopNavigator {
     // Otherwise it is a geometry step and we push the point to the boundary.
     out_state.SetBoundaryState(true);
 
+    if (step < 0.) {
+      step = 0.;
+    }
+
```

Comment:
How can this ever happen? `valid` above checks `ddistance >= 0`

Reply:
Indeed, fixed